LocationManagerExample/LocationManagerExample/INTUViewController.m

```diff
@@ -161,6 +161,7 @@ - (IBAction)forceCompleteRequest:(id)sender
         self.locationRequestID = NSNotFound;
         self.statusLabel.text = @"Subscription canceled.";
     }
+    [self.activityIndicator stopAnimating];
```

Comment:
Is there a reason that you found this to be necessary? When we set `NSNotFound` to the `locationRequestID` property on Line 161, the overridden setter for that property ends up calling `[self.activityIndicator stopAnimating];` on Line 248.